### PR TITLE
new package: c-script

### PR DESCRIPTION
**Pull request by Freed-Wu**

### CODE REVIEW

File: packages/c/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ryanmjacobs/c
+TERMUX_PKG_DESCRIPTION="Compile and execute C "scripts" in one go!"
```
Comment (Grimler91) [inline packages/c/build.sh:2]: Quotes inside a quoted string needs to be escaped

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Using a single character "c" as package name is generally not a good idea, we should add some suffix to it to avoid confusion. What is the package called in other distros? 

Maybe "c-compiler" would work.

(we had the same discussion/issue for o(-editor) before

File: packages/c/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=2b66d79d0d5c60b8e6760dac734b8ec9a7d6a5e57f033b97086821b1985a870b
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS=bash
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+
+termux_step_make_install() {
+  local bin="$(basename $TERMUX_PKG_HOMEPAGE)"
+  install -D "$bin" -t "$TERMUX_PREFIX/bin"
```
Comment (xtkoba) [inline packages/c/build.sh:15]: Use tabs for indentation in Shell scripts, as per <a href="https://github.com/termux/termux-packages/wiki/Coding-guideline">Coding guideline</a>.

**Review by xtkoba — CHANGES_REQUESTED** [review]
I propose the package name `c-script`.

**Comment by Freed-Wu** [comment]
In homebrew and pacman (AUR), its name is c. Maybe the author of c have other
good name.

**Pull request merged**